app/public/survey.html

```diff
@@ -0,0 +1,233 @@
+        <label for="question10">
+          <h5>You feel more energetic after spending time with a group of people.</h5>
+        </label>
+        <select name="q10" class="form-control" id="question10" required>
+          <option value="" disabled selected>Select an Option</option>
+          <option value="1">1 (Strongly Disagree)</option>
+          <option value="2">2</option>
+          <option value="3">3</option>
+          <option value="4">4</option>
+          <option value="5">5 (Strongly Agree)</option>
+        </select>
+      </div>
+      <button id="submit-btn" form="form" type="submit" class="btn btn-lg btn-block btn-primary">Submit</button>
+    </form>
+    <hr>
+    <footer class="footer">
+      <p><a href="/api/friends">API Friends List</a> | <a href="https://github.com/asajbel/FriendFinder">Github Repo</a></p>
+    </footer>
+  </div>
+  <div id="friendModal" class="modal" tabindex="-1" role="dialog">
+    <div class="modal-dialog modal-lg" role="document">
+      <div class="modal-content">
+        <div class="modal-header">
+          <h3 class="modal-title">Meet Your New Friend</h3>
+          <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+            <span aria-hidden="true">&times;</span>
+          </button>
+        </div>
+        <div class="modal-body" align="center">
+          <h5 id="friendName">Name</h5>
+          <img id="friendPic" class="img-fluid rounded" src="">
+        </div>
+        <div class="modal-footer">
+          <button type="button" class="btn btn-secondary" data-dismiss="modal">Close</button>
+        </div>
+      </div>
+    </div>
+  </div>
+  <script src="https://code.jquery.com/jquery.js"></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.12.3/umd/popper.min.js" integrity="sha384-vFJXuSJphROIrBnz7yo7oB41mKfc8JzQZiCq4NCceLEaO4IHwicKwpJf9c9IpFgh" crossorigin="anonymous"></script>
+  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-beta.2/js/bootstrap.min.js" integrity="sha384-alpBpkh1PFOepccYVYDB4do5UnbKysX5WZXm3XxPqe5iKTfUKjNkCk9SaVuEZflJ" crossorigin="anonymous"></script>
+  <script type="text/javascript">
+  // Question: What does this code do?
+  $("#form").on("submit", function(event) {
+    event.preventDefault();
+    var answers = [];
+    for (var i = 2; i < 12; i++) {
+      answers.push(parseInt(event.target[i].value));
+    }
+
+    var newFriend = {
+      "name": event.target.name.value.trim(),
+      "photo": event.target.photo.value.trim(),
+      "scores": answers
+    };
+
+    event.target.reset();
+
+    console.log(newFriend);
+
+    $.ajax({
+      url: "/api/friends",
+      data: JSON.stringify(newFriend),
+      type: "POST",
+      contentType: "application/json",
+      dataType: "json"
+    }).success(function(data) {
+      $("#friendName").text(data.name);
+      $("#friendPic").attr("src", data.photo);
+      $("#friendModal").modal('toggle');
+    });
+
+
+
+  });
+  </script>
```

Comment:
It is good that you know how to perform these functions using the jquery library, however, I would challenge you to do it without the assistance of jquery. In this instance, you could surround your input fields in a form and add action and method to the form and as long as you have body-parser then your post route will receive that data on submission.